vector-im/element-android
# [Poll] When a poll is ended, use /relations API to ensure poll results are correct (PSG-1013)

## Type of change


## Content


When receiving a poll end event, the /relations API is used to fetch all related events so that we ensure to aggregate all the poll responses for this poll. It is performed so that correct results are displayed to the user.

## Motivation and context


Closes #7767 

## Screenshots / GIFs





## Tests



- Create a poll
- Vote from different users
- End the poll
- Check the results of the poll are correct
- Check with flipper correct network calls are done on end poll event reception

## Tested devices

- OS version(s): Android 11

## Checklist
